dropwizard-auth/src/main/java/io/dropwizard/auth/Demo.java

```diff
@@ -0,0 +1,38 @@
+import javax.crypto.Cipher;
+import javax.crypto.NoSuchPaddingException;
+import java.security.NoSuchAlgorithmException;
+
+public class Demo {
+    public static void main(String[] args) throws Exception {
+        String rpository = "test";
```

Comment:
Possible typo in `rpository`, fixes:
Candidate #1 with a confidence 100%
```suggestion
        String repository = "test";
```

<!-- lookout footnote separator -->
_Comment made by the analyzer style-check._